### ISSUE TITLE
Possible backward compatibility issue - SenderPipe does not have forward "failure" anymore

### ISSUE THREAD

**Issue by mhdirkse** [ibissource/iaf#2724]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
Section "Getting Started" of the Frank!Manual contained a configuration with a SenderPipe that contained a FixedQuerySender. The SenderPipe used a forward with name "failure". In the Frank!Doc, this name is not among the forwards of the SenderPipe. I was keen to test the configurations that I put in the Frank!Manual, so I suppose that the SenderPipe had forward "failure" in the past but does not have it now. This may be a backward compatibility issue.

**Reporter**
Martijn Dirkse

**To Reproduce**

**Screenshots**

**Configuration**
```
Please provide the configuration of the Pipe or Receiver with the problem.
```

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```

**Environment**
```
Please go to the console of your Frank, click on 'Information' and copy-paste the results here. 
```

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql, PostgreSQL]
 - Browser: [e.g. Chrome, Safari, Edge, Firefox]

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

**Comment by gvanbrakel** [comment]
Are you sure that the forward named "failure" did actually work?

**Comment by mhdirkse** [comment]
I have to admit: no. Forward "failure" was part of an unhappy flow of my adapter. I am quite certain that I tested the happy flow of my adapter, and I did test a few unhappy flows. But I may have skipped a few unhappy flows, including the "failure" forward of the SenderPipe.

To test this, I can checkout an old version of the F!F that was created by the autumn of 2019. Then I can take my adapter from the Git history of the Frank!Manual and test for example with a corrupt database. I am not so keen to do this, because I have a lot of different tasks. Alternatively, we can ask via the chat or Teams whether our integration specialists use forward "failure" at the moment.
